subcommittees/trainers/README.md

```diff
@@ -5,22 +5,14 @@
 This is the group of certified instructor trainers. We certify new instructors and collaborative develop the instructor training cirriculum.
 
 ## What We Do
-
-### Teach and Develop the Instructor Training Curriculum
-
-See the lesson here: http://swcarpentry.github.io/instructor-training/
-
-### Hold two group meetings a month 
-
-We hold two meetings a month:
-- Disscussion meetings - 1st Wednesday of the month
-- Business meetings - 3rd Wednesday of the month
-
-The etherpad is here: http://pad.software-carpentry.org/trainers
-
-## Host demo sessions for instructor checkout
-
-The etherpad is here: http://pad.software-carpentry.org/teaching-demos
+What we do is described in detail in the instructor trainer agreement policy document: https://github.com/carpentries/policies/blob/master/trainer-agreement.md
+
+Befiefly, we carry out tasks including:
+- develop and teach the Instructor Training Curriculum: http://swcarpentry.github.io/instructor-training/
+- host/attend two group meetings a month: http://pad.software-carpentry.org/trainers
+  - Disscussion meetings - 1st Wednesday of the month
+  - Business meetings - 3rd Wednesday of the month
+- host demo sessionsfor instructor checkout: http://pad.software-carpentry.org/teaching-demos
```

Comment:
- missing space "sessions for"
- I would avoid putting dates for the meetings and just link to the community calendar in case the dates change
- "discussion" (3 s, not 4) :-)